py-utils/src/utils/msg_bus/message_bus/bus/bus_frame.py

```diff
@@ -0,0 +1,142 @@
+import logging
+log = logging.getLogger(__name__)
```

Comment:
Remove all the logging code and Use already existing logger from cortx-utils

Reply:
removed old loggers